src/business-logic/services/rundown-lock-service.ts

```diff
@@ -0,0 +1,85 @@
+import { Part } from '../../model/entities/part'
+import { Piece } from '../../model/entities/piece'
+import { Owner } from '../../model/enums/owner'
+import { LockRundownException } from '../../model/exceptions/lock-rundown-exception'
+import { InTransition } from '../../model/value-objects/in-transition'
+import { RundownService } from './interfaces/rundown-service'
+
+const RUNDOWN_LOCK_INTERVAL_MS: number = 500
+const RUNDOWN_LOCK_ERROR_TEXT: string = 'Unable to do action. An action was already executed less than 500ms ago'
```

Comment:
Should it be more fuzzy - like 'half a second' or even 'less than a second' ago instead of engineering precision in three places after the decimal millis?

Reply:
@KvelaGorrrrnio I fix your comment, @k-primeholding-dim we can do it but I don't see much benefits of that. My idea for this error is to tell the user, please don't spam. If we want we can calculate seconds from this MS and the message will looks like:
`Unable to do action. An action was already executed less than 0.5 seconds ago`